nuxeo-services/login/nuxeo-platform-login-digest/src/main/java/org/nuxeo/ecm/ui/web/auth/digest/DigestAuthenticator.java

```diff
@@ -147,21 +148,74 @@ public void initPlugin(Map<String, String> parameters) {
     }
 
     public static Map<String, String> splitParameters(String auth) {
-        Map<String, String> map = new HashMap<>();
-        try (CSVParser reader = new CSVParser(new StringReader(auth), CSVFormat.DEFAULT)) {
-            Iterator<CSVRecord> iterator = reader.iterator();
-            if (iterator.hasNext()) {
-                CSVRecord record = iterator.next();
-                for (String itemPairStr : record) {
-                    itemPairStr = StringUtils.remove(itemPairStr, QUOTE);
-                    String[] parts = itemPairStr.split(EQUAL_SEPARATOR, 2);
-                    map.put(parts[0].trim(), parts[1].trim());
-                }
-            }
-        } catch (IOException e) {
+        Map<String, String> map;
+        try {
+            map = org.apache.tomcat.util.http.parser.Authorization.parseAuthorizationDigest(new StringReader(auth));
```

Comment:
Should we take care to close the `StringReader`?

Reply:
It just holds a `String` so closing it doesn't bring anything.